10_integrations/algolia_indexer.py

```diff
@@ -73,7 +73,8 @@
 
 
 @stub.function(
-    image=algolia_image, secrets=[Secret.from_name("algolia-secret")]
+    image=algolia_image,
+    secrets=[Secret.from_name("algolia-secret", environment_name="main")],
```

Comment:
i think this might be an issue with the synthetic monitor code maybe? we should probably use the "main" environment for all examples

Reply:
Hm it's coded to be the `syn-mon-{runtime}-{cloud}` environment right now

Reply:
I thought about it a bit, don't see any harm in changing to `main`

Reply:
ok i remember now! i think another option is to keep synthetic monitors in a separate environment, but in that case we need to move all secrets to that env which is a bit annoying

seems fine to move examples back to main